src/content/blog/anatomy-advanced-ddev-addon.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Diffy: Anatomy of an Advanced DDEV Add-on"
+pubDate: 2024-08-07
+summary: Lessons learned running DDEV on an inexpensive Chromebook.
+modifiedDate: 2024-08-07
+author: Yuri Gerasymov
+featureImage:
+  src: /img/blog/2024/08/dismantled-camera.jpeg
+  alt: Microsoft image creator.
+categories:
+  - TechNotes
+---
+
+[Diffy](https://diffy.website) is a visual regression testing tool that allows you to take screenshots of your website and compare them so you know exactly what pages got changed and where. We decided to build DDEV integration so it is possible to take screenshots from the local website, upload them to Diffy, and then compare them with screenshots from other environments.
+
+The `ddev-diffy` add-on is at [github.com/DiffyWebsite/ddev-diffy](https://github.com/DiffyWebsite/ddev-diffy) and can be installed with the instructions there, including `ddev get DiffyWebsite/ddev-diffy`. It does require a Diffy API key to operate.
+
+Watch [7 mins video](https://www.loom.com/share/a3b750e32581458f9d2271969bba1bb8) or read [documentation](https://docs.diffy.website/features/local-development/ddev-add-on).
+
+Architecturally, integration consists of:
+
+* Docker container with preinstalled node and Chrome 111
+* Node.js app that runs Chrome
+* The app itself is hosted in a separate repository as it is used in other places (like production for the main service itself).
+
+Let’s go through the way integration is built.
+
+## Download node app
```

Comment:
People don't have to take any of these actions, true? Why is there a `docker run -it` here? 

I think you just do a `ddev get DiffyWebsite/diffy` right? If that's the whole thing, then say that.